examples/README.md

```diff
@@ -13,6 +13,14 @@ Machine Learning Challenge Winning Solutions
 | Place         | Competition   | Solution  | Date |
 | ------------- |:------------- | --------- | -----|
 |1st      | [M5 Forecasting - Accuracy](https://www.kaggle.com/c/m5-forecasting-accuracy) | [link](https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/163684) | 2020.6 |
+|3rd      | [M5 Forecasting - Uncertainty](https://www.kaggle.com/c/m5-forecasting-uncertainty) | [link](https://www.kaggle.com/c/m5-forecasting-uncertainty/discussion/166875) | 2020.7 |
+|3rd      | [ALASKA2 Image Steganalysis](https://www.kaggle.com/c/alaska2-image-steganalysis) | [link](https://www.kaggle.com/c/alaska2-image-steganalysis/discussion/168870) | 2020.7 |
+|1st      | [M5 Forecasting - Accuracy](https://www.kaggle.com/c/m5-forecasting-accuracy) | [link](https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/163684) | 2020.6 |
```

Comment:
can you please remove this line? This entry already exists in this table

Reply:
I have made the required changes to the first line of winning solutions. I believe there will be more winning KDD/notebooks. I shall update them sooner.